content/en/blog/2024/go-opentelemetry-io-switch.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Planned Migration for go.opentelemetry.io
+linkTitle: Planned migration for go.opentelemetry.io
+date: 2024-08-22
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Mike Dame](https://github.com/damemi) (Google),
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: 5086
+sig: Go SIG
+---
+
+With contributions from Tyler Yahn and Austin Parker.
+
+## tl;dr
+
+**The app that serves requests to go.opentelemetry.io will be migrating to a new
```

Comment:
No. We need to create a markdown link definition and use it where appropriate -- we probably don't want a link for _every_ occurrence (where we don't have a link, it can be in code font using backticks).